### PR TITLE
Params to turn off next/previous page - and breadcrumb

### PR DESCRIPTION
**Pull request by willwade**
I *think* this is all good! 

re: #136

### CODE REVIEW

**Comment by willwade** [comment]
tweaked to allow 

```
disableBreadcrumb: true
disableNextPrev: true
```

In page params or site params

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -40,12 +40,15 @@
         {{ end }}
 
 
-        {{with ($.Scratch.Get "prevPage")}}
-            <a class="nav nav-prev" href="{{.URL}}" title="{{.Title}}"> <i class="fa fa-chevron-left"></i></a>
-        {{end}}
-        {{with ($.Scratch.Get "nextPage")}}
-            <a class="nav nav-next" href="{{.URL}}" title="{{.Title}}" style="margin-right: 0px;"><i class="fa fa-chevron-right"></i></a>
-        {{end}}
+	 {{$showPrevNext := (and (not .Params.disableNextPrev) (not .Site.Params.disableNextPrev))}}
```
Comment (matcornic) [inline layouts/partials/footer.html]: I think this is equivalent to `(or .Params.disableNextPrev .Site.Params.disableNextPrev)`. But I'm not sure this should be the condition to use.

Actually, I think page params should always override behaviour of site params when set. For example, we have a site and we choose to disable all nextprev for the site in the `config.toml`, but we want to enable it on a single page tough. It's not possible with this implementation.

File: layouts/partials/header.html [hunk 2]
```diff
@@ -62,11 +62,16 @@
...
                     </span>
-                  {{ if $toc }}
-                  <span id="toc-menu"><i class="fa fa-list-alt"></i></span>
-                  {{ end }}
+					  {{ if $toc }}
+					  <span id="toc-menu"><i class="fa fa-list-alt"></i></span>
+					  {{ end }}
                   <span class="links">
-                    {{ template "breadcrumb" dict "page" . "value" .Title }}   
+                 {{$showBreadcrumb := (and (not .Params.disableBreadcrumb) (not .Site.Params.disableBreadcrumb))}}
```
Comment (matcornic) [inline layouts/partials/header.html]: same thing

**Comment by matcornic** [comment]
Sorry, I'm super late. See my comments :)

**Comment by matcornic** [comment]
And btw, could you update the documentation (English part, I will do the french one ^^) with your changes ? 

- Site params: http://learn.netlify.com/en/basics/configuration/#global-site-parameters
- Page params: http://learn.netlify.com/en/cont/pages/#front-matter-configuration 

Thanks again :)

**Comment by matalo33** [comment]
Can we merge this in as a sitewide setting and add per-page override to the backlog? I've tried implementing this but can't solve without having a `disableNextPrev` and a `enableNextPrev`.

**Pull request closed without merging**